### PR TITLE
changes on footer

### PR DESCRIPTION
**Pull request by Gunkev**
In this PR

old-footer

# Changes

footer

### CODE REVIEW

**Comment by AemieJ** [comment]
I really prefer the previous footer, it looks minimal and goes with the website design.

**Comment by psingh587** [comment]
Agreed with @AemieJ

**Comment by Gunkev** [comment]
# Changes

Screenshot from 2020-03-31 17-00-51

File: antora-ui-camel/src/css/footer.css [hunk 1]
```diff
@@ -52,6 +52,12 @@ footer .footer dl dt {
   margin-bottom: 0.5rem;
 }
 
+.stroke {
```
Comment (zregvart) [inline antora-ui-camel/src/css/footer.css:55]: Can we scope this to footer?

```suggestion
footer .footer .stroke {
```

Comment (Gunkev) [reply antora-ui-camel/src/css/footer.css:55]: ok

**Comment by zregvart** [comment]
Thanks!

**Comment by zregvart** [comment]
For future reference please run the checks `yarn checks` this broke the website build and needed to be reworked in d8909b7f24bc49f5f9e9f94ed2d2d3a9d42f5ddf.

**Comment by Gunkev** [comment]
> For future reference please run the checks `yarn checks` this broke the website build and needed to be reworked in [d8909b7](https://github.com/apache/camel-website/commit/d8909b7f24bc49f5f9e9f94ed2d2d3a9d42f5ddf).

Understood @zregvart

**Pull request merged**